File: aifupan_project/fupan-server/replay-generic/src/main/java/com/jiuyu/replay/generic/vo/words/SentenceMarkVo.java
package com.jiuyu.replay.generic.vo.words;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.util.List;

@Data
@Schema(description = "文字内容")
public class SentenceMarkVo {

    /**
     * 视频id
     */
    @Schema(description = "视频id")
    private String videoId;
    /**
     * 当前是第几段，从1开始
     */
    @Schema(description = "当前是第几段，从1开始")
    private Integer currentSort;
    /**
     * 文字内容
     */
    @Schema(description = "文字内容")
    private String content;
    /**
     * 词语列表
     */
    @Schema(description = "词语列表")
    private List<WordListItemVo> items;
    /**
     * 关键词/敏感词列表
     */
    @Schema(description = "关键词/敏感词列表")
    private List<WordsMarkVo> wordsList;
}

File: aifupan_project/ReviewAnalysis/Asr/SentenceMarkVo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json;

namespace ReviewAnalysis.Asr
{
    public class SentenceMarkVo
    {
        /// <summary>
        /// 视频id
        /// </summary>
        [JsonProperty("videoId")]
        public string VideoId { get; set; }

        /// <summary>
        /// 当前是第几段，从1开始
        /// </summary>
        [JsonProperty("currentSort")]
        public int CurrentSort { get; set; }

        /// <summary>
        /// 文字内容
        /// </summary>
        [JsonProperty("content")]
        public string Content { get; set; }

        /// <summary>
        /// 词语列表
        /// </summary>
        [JsonProperty("items")]
        public List<WordListItemVo> Items { get; set; }

        /// <summary>
        /// 关键词/敏感词列表
        /// </summary>
        [JsonProperty("wordsList")]
        public List<WordsMarkVo> WordsList { get; set; }
    }
}
